sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway-v2.bpmn

```diff
@@ -210,7 +210,7 @@ Note: the workflow process should continue its execution as a regular reassessme
e: calculatedDataTotalSecondResidence,&#10;  otherAllowableCost: calculatedDataTotalChildSpousalSupport + calculatedDataTotalSecondResidence + calculatedDataTotalAdditionalTransportationAllowance + calculatedDataPartnerStudentLoans,&#10;  transportationCost: calculatedDataTotalTransportationCost,&#10;  secondResidenceCost: calculatedDataTotalSecondResidence,&#10;  totalAssessmentNeed: calculatedDataTotalAssessedNeed,&#10;  booksAndSuppliesCost: calculatedDataTotalBookCost,&#10;  totalProvincialAward: awardNetProvincialTotalAward,&#10;  alimonyOrChildSupport: calculatedDataTotalChildSpousalSupport,&#10;  federalAssessmentNeed: calculatedDataFederalAssessedNeed,&#10;  exceptionalEducationCost: offeringExceptionalExpenses,&#10;  provincialAssessmentNeed: calculatedDataProvincialAssessedNeed,&#10;  totalFederalContribution: calculatedDataTotalFederalContribution,&#10;  parentAssessedContribution: calculatedDataTotalParentalContribution,&#10;  partnerAssessedContribution: calculatedDataTotalSpouseContribution,&#10;  totalProvincialContribution: calculatedDataTotalProvincialContribution,&#10;  studentTotalFederalContribution: calculatedDataTotalFederalFSC,&#10;  totalFederalAssessedResources: parentAssessedContribution + partnerAssessedContribution + calculatedDataTargetedResources + calculatedDataTotalFederalFSC,&#10;  totalProvincialAssessedResources: parentAssessedContribution + partnerAssessedContribution + calculatedDataTargetedResources + calculatedDataTotalProvincialFSC,&#10;  studentTotalProvincialContribution: calculatedDataTotalProvincialFSC,&#10;  finalFederalAwardNetCSGPAmount: finalFederalAwardNetCSGPAmount,&#10;  finalFederalAwardNetCSGDAmount: finalFederalAwardNetCSGDAmount,&#10;  finalProvincialAwardNetBCAGAmount: finalProvincialAwardNetBCAGAmount,&#10;  finalProvincialAwardNetSBSDAmount: finalProvincialAwardNetSBSDAmount,&#10;  finalProvincialAwardNetBCSLAmount: finalProvincialAwardNetBCSLAmount,&#10;  finalFederalAwardNetCSGFAmount: finalFederalAwardNetCSGFAmount,&#10;  finalProvincialAwardNetBGPDAmount: finalProvincialAwardNetBGPDAmount,&#10;  finalAwardTotal: awardNetFederalTotalAward + awardNetProvincialTotalAward,&#10;  studentExpectedFederalContribution: calculatedDataTotalTargetedResources + calculatedDataTotalFederalFSC,&#10;  studentExpectedProvincialContribution: calculatedDataTotalTargetedResources + calculatedDataTotalProvincialFSC,&#10;  scholarshipsBursariesContribution: calculatedDataTotalScholarshipsBursaries,&#10;  targetedResources: calculatedDataTargetedResources,&#10;  partnerExpectedContribution: calculatedDataTotalSpouseContribution,&#10;  parentalDiscretionaryContribution: calculatedDataParentalDiscretionaryContribution,&#10;  parentalAssetContribution: calculatedDataParentalAssetContribution,&#10;  parentalVoluntaryContribution: calculatedDataParentalContribution,&#10;  exceptionalCosts: calculatedDataExceptionalCosts,&#10;  partnerStudentLoans: calculatedDataPartnerStudentLoans&#10;}" target="assessmentData" />
```

Comment:
Same for second residence. 

<img width="826" height="148" alt="image" src="https://github.com/user-attachments/assets/fddd37ca-eaee-4846-ac1b-8ab36e6deccc" />